mysite/ct/views.py

```diff
@@ -772,7 +773,10 @@ def _lessons(request, pageData, concept=None, msg='',
             s = searchForm.cleaned_data['search']
             if searchType is None:
                 searchType = searchForm.cleaned_data['searchType']
-            lessonSet = UnitLesson.search_text(s, searchType)
+            lessonSet = UnitLesson.search_text(s, searchType, dedupe=False).exclude(unit=unit)
+            treeIDs_head = distinct_subset(lessonSet)
```

Comment:
I would use the variable name `tree_ids_head` instead of camelCase naming as per PEP8 standard